packages/repo-tools/src/postinstall.mts

```diff
@@ -28,21 +28,19 @@ if (process.env.SKIP_POSTINSTALL) {
   process.exit(0);
 }
 
-void (async function (): Promise<void> {
```

Comment:
Shouldn't this be allowed by `allowAsStatement: true`?

Reply:
Yes it should, but it should also be unnecessary because top-level `await` is always allowed in `mts` files, I believe

Reply:
Ok, gotcha, that's what I figured... Still - that means that this change is effectively entirely unrelated (I'm guessing maybe it came about because at first you were testing without `allowAsStatement: true`?). Very pedantic of me, but I'd prefer we keep it out of this diff for git cleanliness reasons. Feel free to submit it in a separate change!

Reply:
makes sense!